test/Microsoft.AspNetCore.Mvc.Razor.Test/Internal/DefaultRoslynCompilationServiceTest.cs

```diff
@@ -375,5 +405,15 @@ private static ApplicationPartManager GetApplicationPartManager()
                 GetFileProviderAccessor(fileProvider),
                 NullLoggerFactory.Instance);
         }
+
+        private class TestRoslynCompilationService : DefaultRoslynCompilationService
+        {
+            public TestRoslynCompilationService(ApplicationPartManager partManager, RazorViewEngineOptions options)
+                : base(partManager, GetAccessor(options), GetFileProviderAccessor(), NullLoggerFactory.Instance)
+            {
+            }
+
+            public IList<MetadataReference> GetProtectedCompilationReferences() => GetCompilationReferences();
```

Comment:
`GetCompilationReferencesPublic`? I think we have a precedent to use that naming convention elsewhere.